tests/testthat/_snaps/survival-workflow-augment-dynamic.md

```diff
@@ -0,0 +1,18 @@
+# augment survival workflows with eval_time
+
+    Code
+      augment(wflow_fit, new_data = sim_dat)
+    Condition
+      Error in `augment()`:
+      ! The `eval_time` argument is missing, with no default.
+
+---
+
+    Code
+      workflow() %>% add_model(proportional_hazards(penalty = 0.001) %>% set_engine(
+        "glmnet")) %>% add_formula(event_time ~ .) %>% fit(data = sim_dat) %>%
+        augment(new_data = sim_dat)
+    Condition
+      Error in `h()`:
+      ! error in evaluating the argument 'x' in selecting a method for function 'as.matrix': Cholmod error 'X and/or Y have wrong dimensions' at file ../MatrixOps/cholmod_sdmult.c, line 88
```

Comment:
I'm hesitant to check in a snapshot of something we don't want because I think it's easy to lose sight of that, particularly in extratests. I've added a `skip()` to help us remember more easily and to avoid this snapshot going into main :)